src/Collector/ImplementsCollector.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace SensioLabs\Deptrac\Collector;
+
+use PhpParser\Node\Name;
+use PhpParser\Node\Stmt\Class_;
+use PhpParser\Node\Stmt\Interface_;
+use SensioLabs\AstRunner\AstMap;
+use SensioLabs\AstRunner\AstParser\AstClassReferenceInterface;
+use SensioLabs\AstRunner\AstParser\AstParserInterface;
+use SensioLabs\AstRunner\AstParser\NikicPhpParser\NikicPhpParser;
+use SensioLabs\Deptrac\CollectorFactory;
+
+class ImplementsCollector
```

Comment:
Funny how this is missing the `CollectorInterface` then :D

Reply:
:-D And funny is that it works.